net.wz2100.wz2100.yaml

```diff
@@ -0,0 +1,160 @@
+app-id: net.wz2100.wz2100
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: warzone2100
+separate-locales: false
+finish-args: 
+  - --socket=wayland
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --filesystem=home
+  - --socket=pulseaudio
+  - --share=network
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /share/gtk-doc
+  - /share/man
+  - /share/pkgconfig
+  - '*.a'
+  - '*.la'
+  
+modules:
+  - shared-modules/glu/glu-9.0.0.json
+  - shared-modules/glew/glew.json
+
+  - name: qt5-script
+    buildsystem: qmake
+    make-install-args:
+    - INSTALL_ROOT=/app
+    post-install:
+      - find "/app/usr/lib" -type f -name '*.prl' -exec sed -i -e '/^QMAKE_PRL_BUILD_DIR/d' {} \;
+      
+    sources:
+    - type: archive
+      url: 'http://download.qt.io/official_releases/qt/5.11/5.11.0/submodules/qtscript-everywhere-src-5.11.0.tar.xz'
+      sha256: 25926ae733b31baac7af51f489ad26570d4f4f02ad4892a4a82babae5f5168c5
+
+  - name: sdl2
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+    - -DCMAKE_INSTALL_PREFIX=/app
+    - -DSDL_STATIC=OFF
+    - -DSDL_DLOPEN=ON
+    - -DARTS=OFF
+    - -DESD=OFF
+    - -DNAS=OFF
+    - -DALSA=ON
+    - -DPULSEAUDIO_SHARED=ON
+    - -DVIDEO_WAYLAND=ON
+    - -DRPATH=OFF
+    - -DCLOCK_GETTIME=ON
+    sources:
+      - type: archive
+        url: 'https://www.libsdl.org/release/SDL2-2.0.8.tar.gz'
+        sha512: 673c6058b8692a36b4a3594456b10ef6051efe79e4fb644421fc5c76b11fd68b895840a2c8b72413418c378733e2993d33f19767d0d7ed101eda6310bd70c869
+  
+  - name: libogg
+    config-opts:
+    - --prefix=/app
+    - -disable-static
+    sources:
+      - type: archive
+        url: 'https://downloads.xiph.org/releases/ogg/libogg-1.3.3.tar.xz'
+        sha256: 4f3fc6178a533d392064f14776b23c397ed4b9f48f5de297aba73b643f955c08
+        
+  - name: libvorbis
+    config-opts:
+    - --prefix=/app
+    sources:
+      - type: archive
+        url: 'https://downloads.xiph.org/releases/vorbis/libvorbis-1.3.6.tar.gz'
+        sha256: 6ed40e0241089a42c48604dc00e362beee00036af2d8b3f46338031c9e0351cb
+        
+  - name: physfs
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+    - -DCMAKE_INSTALL_PREFIX=/app
+    - -DCMAKE_BUILD_TYPE=Release
+    - -DPHYSFS_BUILD_TEST=OFF
+    sources:
+      - type: archive
+        url: 'http://icculus.org/physfs/downloads/physfs-3.0.1.tar.bz2'
+        sha1: 9959a7acad0aa30d1a86d3a418e61155b0ac533c
+      
+  - name: ffmpeg
+    config-opts:
+      - --disable-everything
```

Comment:
Doesn't it mean you have no decoders at all?

Reply:
No, only need OGG audio codec and Theora video codec. The ingame videos work perfect

Reply:
Then it doesn't use ffmpeg at all and the module can be removed.